cpp/lib/src/ZoteroHarvesterConversion.cc

```diff
@@ -511,6 +511,18 @@ void PostProcessAuthorName(std::string * const first_name, std::string * const l
 }
 
 
+const std::string TIKA_SERVER_DETECT_STRING_LANGUAGE_URL("http://localhost:9998/language/string");
```

Comment:
Installer Changes for Tika are missing.

I wonder - do we really need a server (either as running service or within a docker container), or would it be simpler to use the command line interface?

Reply:
Command line would mean spawning a java process on each call.